src/ray/core_worker/core_worker.cc

```diff
@@ -1176,6 +1206,31 @@ Status CoreWorker::Get(const std::vector<ObjectID> &ids,
   bool got_exception = false;
   absl::flat_hash_map<ObjectID, std::shared_ptr<RayObject>> result_map;
   auto start_time = current_time_ms();
+  std::ostringstream ids_stream;
+
+  for (size_t i = 0; i < ids.size(); i++) {
+    rpc::Address unused_owner_address;
+    auto status = GetOwnerAddress(ids[i], &unused_owner_address);
+    if (!status.ok()) {
```

Comment:
Instead of not ok, can we check ObjectUnknownOwner explicitly?

Reply:
done